src/cbmc_starter_kit/print_tool_versions.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+
+import logging
+import pathlib
+import shutil
+import subprocess
+
+
+_TOOLS = [
+    "cadical",
```

Comment:
Could we also print minisat version since we are printing cadical?

Reply:
I don't think minisat is installed as an executable, CBMC statically links it.

I need some kind of executable command that prints out the version number. I don't think we have anything like this for minisat. One path forward might be to add a flag to CBMC that prints out the minisat version, if this is important